webpack/assets/javascripts/react_app/components/common/CloseButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from './Icon';
+
+const CloseButton = ({onClick}) => {
+  return (
+    <button className="close" aria-hidden="true" onClick={onClick}>
+      <Icon type="close" />
```

Comment:
In this case no. It doesn't in the patternfly example. I think the X is pretty straight forward.